gravitee-apim-gateway/gravitee-apim-gateway-flow/src/main/java/io/gravitee/gateway/reactive/flow/AbstractFlowResolver.java

```diff
@@ -17,22 +17,24 @@
 
 import io.gravitee.definition.model.flow.Flow;
 import io.gravitee.gateway.reactive.api.context.GenericExecutionContext;
+import io.gravitee.gateway.reactive.api.context.base.BaseExecutionContext;
+import io.gravitee.gateway.reactive.api.context.http.HttpBaseExecutionContext;
 import io.gravitee.gateway.reactive.core.condition.ConditionFilter;
 import io.reactivex.rxjava3.core.Flowable;
 
 /**
  * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
  * @author GraviteeSource Team
  */
-public abstract class AbstractFlowResolver implements FlowResolver {
+public abstract class AbstractFlowResolver<T extends BaseExecutionContext> implements FlowResolver<T> {
```

Comment:
just to have a comment :p
```suggestion
public abstract class AbstractFlowResolver<C extends BaseExecutionContext> implements FlowResolver<T> {
```